app.js

```diff
@@ -8,5 +59,70 @@ function closeMain() {
   main.style.visibility = 'hidden';
 }
 
+function closeModel() {
+  modelView.style.visibility = 'hidden';
+}
+
+function openModel(id) {
+  modelView.style.visibility = 'visible';
+  for (let i = 0; i < array.length; i += 1) {
+    if (id === array[i].id) {
+      modelView.innerHTML = `
+      <div class="model_content" id="modelcontent">
+      <div class="model_header">
+          <h2 class="project_title">${array[i].name}</h2>
+          <span onclick="closeModel()" class="crossBtn">&times;</span>
+      </div>
+      <ul class="info_list">
+          <li class="item1">${array[i].featured[0]}</li>
+          <li class="item2"><span class="dot">&bull;</span><span>${array[i].featured[1]}</span></li>
+          <li class="item2"><span class="dot">&bull;</span><span>${array[i].featured[2]}</span></li>
+      </ul>
+      <img class="img_project" src=${array[i].image} alt="project">
+      <img class="img_desktop" src=${array[i].imageDesktop} alt="project desktop screenshot">
+      <p class="project_description">${array[i].longDescription}</p>
+      <ul class="tech_list">
+          <li>${array[i].technologies[0]}</li>
+          <li class="margin_li">${array[i].technologies[1]}</li>
+          <li class="margin_li">${array[i].technologies[2]}</li>
+      </ul>
+      <div class="buttons">
+          <a class="model_btn live" href=${array[i].live_version}>See live<img class="_icon" src="./images/Icon.svg" alt="live icon"></a>
+          <a class="model_btn source" href=${array[i].source_link}>See project<img class="_icon" src="./images/Vector.svg" alt="github icon"></a>
+      </div>
```

Comment:
- [ ] On the mobile design popup, there is a light border between the tags and the button. This is not reflected in your design. Kindly implement it and adjust the margins accordingly. Check the screenshot below.
- Your design: 
![Screenshot from 2021-12-16 13-44-47](https://user-images.githubusercontent.com/20245164/146358078-e67fd99f-9f72-4b12-b0ff-2f7394769d43.png)
- Given design:
![Screenshot from 2021-12-16 13-44-09](https://user-images.githubusercontent.com/20245164/146358222-02060c09-c105-4996-9c10-ac1aa49024c5.png)